projects/acs-engine/azure-purge/build.yaml

```diff
@@ -0,0 +1,27 @@
+kind: BuildConfig
```

Comment:
We need to rename acs-engine to azure and have an acs-engine subdirectory. Can be done as a follow-up but open an issue to track please.

Reply:
https://github.com/openshift/release/issues/988